db/helpers/seed_helper.rb

```diff
@@ -0,0 +1,29 @@
+class SeedHelper
+  def objectify(model)
+    return model.titleize.constantize unless model.include? "_"
```

Comment:
Can you give it a try in rails console and let m know what you get? Could be helpful.